### PR TITLE
add patch for hard-coded checksum value of downloaded source file in the source code of RDKit 2021.03.4

### PR DESCRIPTION
**Pull request by SIB-software**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/r/RDKit/RDKit-2021.03.4_fix-comic-neue-checksum.patch [hunk 1]
```diff
@@ -0,0 +1,11 @@
+--- rdkit-Release_2021_03_4/Code/GraphMol/MolDraw2D/CMakeLists.txt.orig	2022-01-17 13:56:43.780726092 +0100
```
Comment (boegel) [inline easybuild/easyconfigs/r/RDKit/RDKit-2021.03.4_fix-comic-neue-checksum.patch:1]: @SIB-software A short description should be added on top of this patch, to clarify why this is needed
Author or upstream reference for this change should also be mentioned.

Note that this implies changing the SHA256 checksum for this patch file too...

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by robinengler** [comment]
Thanks @boegel for reviewing this PR.

Would the following description for the patch be OK ? (or is it too long)

```
As part of its installation, RDKit downloads a font package from https://fonts.google.com
and checks its md5sum. It appears the font package was modified and therefore a new md5sum
must be used. The authors of RDKit have updated the md5sum in the source code of RDKit but
no backward fix was made for already existing releases. This patch updates the md5 checksum
to the its new value.
Author: Robin Engler, SIB Swiss Institute for Bioinformatics
--- rdkit-Release_2021_03_4/Code/GraphMol/MolDraw2D/CMakeLists.txt.orig	2022-01-17 13:56:43.780726092 +0100
+++ rdkit-Release_2021_03_4/Code/GraphMol/MolDraw2D/CMakeLists.txt	2022-01-17 13:57:02.668646178 +010
...
```

**Comment by boegel** [comment]
> Thanks @boegel for reviewing this PR.
> 
> Would the following description for the patch be OK ? (or is it too long)
> 
> ```
> As part of its installation, RDKit downloads a font package from https://fonts.google.com
> and checks its md5sum. It appears the font package was modified and therefore a new md5sum
> must be used. The authors of RDKit have updated the md5sum in the source code of RDKit but
> no backward fix was made for already existing releases. This patch updates the md5 checksum
> to the its new value.
> Author: Robin Engler, SIB Swiss Institute for Bioinformatics
> --- rdkit-Release_2021_03_4/Code/GraphMol/MolDraw2D/CMakeLists.txt.orig	2022-01-17 13:56:43.780726092 +0100
> +++ rdkit-Release_2021_03_4/Code/GraphMol/MolDraw2D/CMakeLists.txt	2022-01-17 13:57:02.668646178 +010
> ...
> ```

@robinengler Looks perfect to me

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2684.swalot.os - Linux UNKNOWN UNKNOWN, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/0e338d1ac92cc89b09f12c3086f49642 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @SIB-software!

**Pull request merged**